beets/autotag/hooks.py

```diff
@@ -163,7 +163,7 @@ def __init__(self, title=None, track_id=None, release_track_id=None,
                  composer=None, composer_sort=None, arranger=None,
                  track_alt=None, work=None, mb_workid=None,
                  work_disambig=None, bpm=None, initial_key=None, genre=None,
-                 **kwargs):
+                 trackdisambig=None, **kwargs):
```

Comment:
This addition is actually not necessary anymore, now that we use `kwargs` as a catch-all for arbitrary field names. The list here of old field names is just for backwards-compatibility.